### PR TITLE
Currency branch

### PR DESCRIPTION
**Pull request by allison-grothman**
Implemented:
- Collect salary if player passes payday
- Collect double salary if player lands on payday
- Give player the choice of an initial career if they choose the bootcamp path

To-do:
- Add calls to Flask endpoints to update player {career, salary, cash}

### CODE REVIEW

**Comment by taylor-noel** [comment]
Looks good! Could you please remove the salary and career from the initial player properties (it still shows 3000 as salary and barista as job if they choose the university path) and deduct the 100000 from cash if the player chooses university

**Comment by allison-grothman** [comment]
Both done!

**Comment by taylor-noel** [comment]
I think there may still be a bug. When I choose university now it stays on the Bootcamp path and when I choose BootCamp and my job the career changes but the salary doesn't

File: package-lock.json
Comment (taylor-noel) [inline package-lock.json:1]: this file shouldn't be committed

**Comment by allison-grothman** [comment]
Alrighty, ready to merge

**Pull request merged**